docs/Data_Portal/Users_Guide/gene_expression_clustering.md

```diff
@@ -121,72 +110,53 @@ The control panel as shown has various functionalities with which users can chan
 
 The clustering control button provides several options to modify the default clustering of the heatmap. Click on the button labeled 'Clustering' to display a menu with options as shown.
 
-[![Clustering button](./images/geneexpclust/4-clustering-control.png)](./images/geneexpclust/4-clustering-control.png 'Click to see the full image.')
+[<img src="./images/geneexpclust/4-clustering-control.png" width="600"/>](./images/geneexpclust/4-clustering-control.png 'Click to see the full image.')
```

Comment:
all the images didn't load for me -- see the other PR that was closed (https://github.com/NCI-GDC/gdc-docs/pull/514) for specifics.

Reply:
I haven't tested this yet, but I think the issue is that it's sort of a hybrid HTML-markdown code.